tmt/steps/__init__.py

```diff
@@ -167,11 +176,15 @@ class (single class or tuple of classes).
                 if classes is None or isinstance(plugin, classes)],
             key=lambda plugin: plugin.order)
 
-    def try_running_login(self):
-        """ Run all loaded Login plugin instances of the step """
+    def actions(self):
+        """ Run all loaded Login or Reboot plugin instances of the step """
         for plugin in self.plugins():
+            if isinstance(plugin, Reboot):
+                plugin.go()
+                return
```

Comment:
Why do you need to return here? That doesn't make much sense to me, you want to try to run all of the plugins if it is requested, don't you? There may be 2 login plugins for one step (e.g. when using `tmt run -a login --step report:30 --step report:80`) in different phases. This way you would only run one of them.

Reply:
Changed -> solved.